test/unit/spec/EnrollSms_spec.js

```diff
@@ -353,7 +339,7 @@ define([
           .then(function () {
             expect($.ajax.calls.count()).toBe(2);
             Expect.isJsonPost($.ajax.calls.argsFor(0), {
-              url: 'https://foo.com/idp/idx/introspect',
+              url: 'https://foo.com/api/v1/authn',
```

Comment:
how come there was no authn call before ?

Reply:
We'hv made change in the refreshauthController that enables our V1 tests to call it directly with a stateToken which eventually calls `api/v1/authn`, thats what this change is reflecting.

Reply:
@nikhilvenkatraman-okta  can you point me to the original change. Like you said the request seems to go to authn. Has it always been the case or a recent change ?

Reply:
So back when we made the IDX changes we adjusted this test. But after that we made [this](https://github.com/okta/okta-signin-widget/pull/905/files) fix which basically enables our tests to hit refresh auth with a token